h2o-genmodel/src/main/java/hex/genmodel/attributes/parameters/KeyValue.java

```diff
@@ -1,5 +1,7 @@
 package hex.genmodel.attributes.parameters;
 
+import java.util.Comparator;
+
 public class KeyValue {
```

Comment:
Does this need to be Serializable? Is it used in any Serializable class?

Reply:
We have a test which failed when ContributionComposer did not implement Serializable. In this case it seems ok but its not harm to implement Serializable so I'll do it